pydantic/main.py

```diff
@@ -94,12 +94,18 @@ class BaseConfig:
     arbitrary_types_allowed = False
     json_encoders: Dict[AnyType, AnyCallable] = {}
     orm_mode: bool = False
+    alias_generator: Optional[Callable[[str], str]] = None
 
     @classmethod
     def get_field_schema(cls, name: str) -> Dict[str, str]:
         field_config = cls.fields.get(name) or {}
         if isinstance(field_config, str):
             field_config = {'alias': field_config}
+        elif cls.alias_generator:
```

Comment:
```suggestion
        elif not field_config and cls.alias_generator:
```

So if you set an alias with

```py
class Config:
    fields = {'foobar': {'alias': 'Foobar'}}
```
It still works.

Reply:
Np, thanks for pointing that out! I think I should write a test for it.

Reply:
great, yes please.

Reply:
Done